packages/fxa-content-server/app/scripts/views/sign_in_password.js

```diff
@@ -96,6 +97,7 @@ const SignInPasswordView = FormView.extend({
     const hasLinkedAccountAndNoPassword = hasLinkedAccount && !hasPassword;
     context.set({
       email: account.get('email'),
+      nimbusExperiments: Nimbus.experiments,
```

Comment:
Should we just do this inside the `fxa-settings` package instead? Just a drive-by comment since I'm not sure what the plan is on the A/A test, but if we want this in Backbone then we may want this on a different page. With the next sprint close, are hoping to have the React version of this flow rolled out to 100% of users. If we don't want to do email-first (index/home screen), the only other good options appear to be `post_verify/finish_account_setup/set_password`, `post_verify/password/force_password_change` (which probably doesn't get a ton of traffic), `/pair`, or `/connect_another_device`.

Reply:
@LZoog Sorry for the slow response, I didn't see this comment earlier!

> Should we just do this inside the `fxa-settings` package instead? Just a drive-by comment since I'm not sure what the plan is on the A/A test, but if we want this in Backbone then we may want this on a different page. With the next sprint close, we are hoping to have the React version of this flow rolled out to 100% of users.

We might not run an A/A experiment on this page for the reasons you mentioned (low traffic and React roll-out). 

Adding it in fxa-settings was our first desire, but we found it was easier to implement this in Backbone first. We ran into the case where we want to stop-the-world and have the experiments fetched before we do the first render of settings. What we have in this branch is a neat idea that Dan had tried but it doesn't seem to stop the first render from happening. When you get a chance, some feedback on how to do this in React would be super helpful: https://github.com/mozilla/fxa/commit/50013ed0527ced1b22c762902a4fac30ef897c86#diff-fcaf5b120aa0a31ed6008ef78ae18a42d4bfa76ba21eef30a622219255ecaa2d

As for this patch, I can remove the experiments from the sign-in page for now and re-consider if those other ones are worth running the experiment on later if we can't find a good solution for React sooner.

Reply:
@jonalmeida No problem and makes sense!

I pinged with him briefly yesterday - we may want to add a new `useEffect` for the call in `App.tsx` and [make sure it completes](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/App/index.tsx#L182) alongside these other required states. I think this will work but looking at that branch, I'm not sure if we are generating a `uniqueUserId` for all users if they just don't have that set in local storage yet, since I'm not sure how that'd work if they sign in as a different user. If you want to start by targeting only signed in users where we have access to the UID, you could start by setting it up in `Settings` instead around the place where we do the initial GQL query to populate `/settings`.